harvard-leeds-beckett-university.csl

```diff
@@ -0,0 +1,644 @@
+      <text variable="locator" />
+    </else>
+  </choose>
+</macro>
+
+<macro name="publisher">
+  <choose>
+    <if type="webpage">
+      <group delimiter=": ">
+        <text variable="publisher-place" />
+        <text variable="container-title" />
+      </group>
+    </if>
+    <else>
+      <group delimiter=": ">
+        <text variable="publisher-place" />
+        <text variable="publisher" />
+      </group>
+    </else>
+  </choose>
+</macro>
+
+<macro name="title">
+  <choose>
+    <if type="book  legal_case  legislation  map  musical_score  motion_picture  patent  broadcast  report  song  thesis  webpage  bill">
+      <text variable="title" text-case="title" font-style="italic" />
+    </if>
+    <else>
+      <text variable="title" text-case="title" />
+    </else>
+  </choose>
+</macro>
+
+<macro name="volume">
+  <choose>
+    <if variable="volume">
+      <text term="volume" form="short" suffix=" " />
+      <number variable="volume" />
+    </if>
+  </choose>
+</macro>
+
+<macro name="volume-edition">
+  <choose>
+    <if type="entry-dictionary  entry-encyclopedia">
+      <group delimiter=", ">
+        <text macro="volume" />
+        <text macro="edition" />
+      </group>
+    </if>
+    <else>
+      <group delimiter=". ">
+        <text macro="volume" />
+        <text macro="edition" />
+      </group>
+    </else>
+  </choose>
+</macro>
+
+<macro name="year">
+  <choose>
+    <if variable="issued">
+      <date variable="issued">
+        <date-part name="year" />
+      </date>
+    </if>
+    <else>
+      <text term="no date" form="short" />
+    </else>
+  </choose>
+</macro>
+
+
+<citation disambiguate-add-names="true" disambiguate-add-givenname="true" givenname-disambiguation-rule="all-names-with-initials" disambiguate-add-year-suffix="true">
+  <layout prefix="(" suffix=")" delimiter="; ">
+    <group delimiter=", ">
+      <choose>
+        <if variable="composer">
+          <names variable="composer">
+            <name form="short" initialize-with=". " name-as-sort-order="all" delimiter=", " delimiter-precedes-et-al="never" delimiter-precedes-last="never" and="symbol" et-al-min="3" et-al-use-first="1" />
+          </names>
+        </if>
+        <else-if variable="author">
+          <names variable="author">
+            <name form="short" initialize-with=". " name-as-sort-order="all" delimiter=", " delimiter-precedes-et-al="never" delimiter-precedes-last="never" and="symbol" et-al-min="3" et-al-use-first="1" />
+          </names>
+        </else-if>
+        <else-if variable="editor">
+          <names variable="editor">
+            <name form="short" initialize-with=". " name-as-sort-order="all" delimiter=", " delimiter-precedes-et-al="never" delimiter-precedes-last="never" and="symbol" et-al-min="3" et-al-use-first="1" />
+          </names>
+        </else-if>
+        <else-if type="webpage  entry-encyclopedia  entry-dictionary  post-weblog">
```

Comment:
you need  ```match="any"``` here. ```if``` conditions default to ```match="all"``` which can obviously never be true for multiple types.